frontend/src/modules/entitydetails/components/EntityDetails.js

```diff
@@ -168,6 +169,17 @@ export class EntityDetailsBase extends React.Component<InternalProps, State> {
         dispatch(machinery.actions.get(source, locale, pk));
     }
 
+    copyLinkToClipboard = () => {
+        const { locale, project, resource, entity } = this.props.parameters;
+
+        const string_link = `${location.protocol}//${location.host}/${locale}/${project}/${resource}/?string=${entity}`
```

Comment:
Sorry for misleading you, you actually have to use `window.location...` instead of `location...`, otherwise eslint fails.

Please also keep line length under 100 characters. You could do something like this:
```js
const { locale, project, resource, entity } = this.props.parameters;
const { host, protocol } = window.location;

const link = `${protocol}//${host}/${locale}/${project}/${resource}/?string=${entity}`;
```

Please also add comma at the end.